src/components/views/emojipicker/EmojiPicker.tsx

```diff
@@ -267,6 +390,9 @@ class EmojiPicker extends React.Component<IProps, IState> {
                 emojis = this.memoizedDataByCategory[cat.id];
             } else {
                 emojis = cat.id === "recent" ? this.recentlyUsed : DATA_BY_CATEGORY[cat.id];
+                if (cat.id === "custom") {
+                    emojis = this.finalEmotes;
+                }
```

Comment:
Given it just immediately undoes L392 this looks like it should be an `else if` case before this else case